Author: Daniel Novotny <dnovotny@redhat.com>
Description: Add new magic for MDMP crash report data files.

diff -Naurp file.orig/magic/Magdir/misctools file/magic/Magdir/misctools
--- file.orig/magic/Magdir/misctools	2009-12-05 17:34:07.000000000 +0100
+++ file/magic/Magdir/misctools	2009-12-05 17:35:01.000000000 +0100
@@ -25,3 +25,6 @@
 # Extension: .lo
 # Submitted by: Abel Cheung <abelcheung@gmail.com>
 0	search/80	.lo\ -\ a\ libtool\ object\ file	libtool object file
+
+# From: Daniel Novotny <dnovotny@redhat.com>
+0	string		MDMP\x93\xA7				MDMP crash report data
